src/test/java/io/vertx/test/core/HttpTest.java

```diff
@@ -3179,6 +3255,7 @@ public void testTLSClientRevokedServerCert() throws Exception {
 
   @Test
   //Client specifies cert that the server does not trust via a revoked certificate of the CA
+  @Ignore // this test must be reviewed - it was "passing" when "fail" was expected
```

Comment:
So where does this leave us? :-) Will you fix this @davidafsilva or do you want me to take a look?